tests/helpers/include/params.h

```diff
@@ -165,6 +179,20 @@ MATH_CASE(Bfp8_b)
 
 #undef MATH_CASE
 
+
+#if !defined(MATH_BFP8_B) && !defined(MATH_INT32) && !defined(MATH_FLOAT32) && !defined(MATH_FLOAT16) && !defined(MATH_FLOAT16_B)
+constexpr bool is_fp32_dest_acc_en = dest_acc || format_combo_is_outlier(UNPACK_A_IN, PACK_OUT, dest_acc);
+constexpr Formats pipeline_formats = get_data_formats(UNPACK_A_IN, PACK_OUT, dest_acc);
```

Comment:
```suggestion
constexpr Formats pipeline_formats = get_data_formats(UNPACK_A_IN, PACK_OUT, is_fp32_dest_acc_en);
```
Shouldn't it be like this now?

Reply:
I understand your confusion...Some format combinations are not supported when dest accumulation is off, so we have to turn dest accumulation on. In order to avoid doing this check and changing every cpp test file, instead I define an intermediate value `dest_acc` that holds the flag for dest accumulation according to the python test setup. Then use this value and check if dest accumulation is off and the combination is an outlier which means dest accumulation needs to be turned on. 
```cpp
constexpr bool is_fp32_dest_acc_en = dest_acc || is_format_combination_outlier(UNPACK_A_IN, PACK_OUT, dest_acc);
```
Since `is_fp32_dest_acc_en` is conditionally defined and the argument for dest accumulation in llk function is passed in as a constant, so we can't redefine it, we must use `dest_acc` in order to define `is_fp32_dest_acc_en` once.

Reply:
Yes sure!